01-reading.Rmd

```diff
@@ -17,8 +17,8 @@ By the end of the chapter, students will be able to:
 - define the following:
     - absolute file path
     - relative file path
-    - url
-- read data into R using a relative path and a url
+    - URL
```

Comment:
Is the first time we see the acronym URL? If so, should we define it here, or should we do that in the main text? I am thinking here and in the main text...

Reply:
agree -- both!